src/utils.ts

```diff
@@ -36,12 +38,24 @@ export function formatAddress(address: Address, lang: string): string {
   return `${streetName} ${streetNumber}, ${postalCode} ${city}`;
 }
 
+export function formatAddresses(permits: Permit[], lang: string): string {
+  const addresses: string[] = [];
+  permits.forEach(permit => {
+    if (permit?.address) {
+      addresses.push(formatAddress(permit.address, lang));
+    }
+  });
+  return addresses.length > 0 ? addresses.join(', ') : '-';
+}
+
 export function formatDateDisplay(datetime: string | Date): string {
+  if (!datetime) return '-';
```

Comment:
Same as the other utility functions, I think it's better to check the validity of the parameter in the function that calls this format function, and let the format function do only the "formatting".